xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/commentsinline.vm

```diff
@@ -279,6 +280,35 @@ $xwiki.ssfx.use('uicomponents/viewers/comments.css', true)
   #end
 #end
 ##
+##
+##
+#macro(showPermalinkModal)
+  <div class="modal fade" id="permalinkModal" tabindex="-1" role="dialog">
+    <div class="modal-dialog">
+      <div class="modal-content">
+        <div class="modal-header">
+          <button type="button" class="close" data-dismiss="modal">&times;</button>
+          <h4 class="modal-title">Permalink</h4>
+        </div>
+        <div class="modal-body">
+            <div class="form-group">
+                  <div class="input-group">
+                    <div class="input-group-addon"></div>
+                    <input type="text" class="form-control"  value="">
```

Comment:
I would close the input with ``/>`` and I would remove the empty value attribute.